### ISSUE TITLE
Disable future 'days'

### ISSUE THREAD

**Issue by pie6k** [christopherdro/react-native-calendar#156]
Is it possible to 'gray-out' all days in the future eg. make them look differently and make them not-clickable?

**Comment by hejiaji** [comment]
Of course, you just need to customize the style of days. and do not do anything on the onDateSelect event. Is that what you want?

**Comment by hejiaji** [comment]
Haha, interesting feature,  @christopherdro do you think this is a necessary feature?

**Comment by pie6k** [comment]
I've got calendar that is responsible for displaying logs. Logs in their nature - are showing only events from the past. In such situation - having calendar allowing future days has no point.

**Comment by jungleyu** [comment]
Yeah, it is very helpful to add date range configuration.

**Comment by peter-mach** [comment]
Agree, I have a use case where user selects previous period date. This is not possible with future dates.
